src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/util/TestResource.java

```diff
@@ -38,4 +46,43 @@ public static Document testDocument() {
             .binaryUrl(BINARY_URL)
             .build();
     }
+
```

Comment:
I like the idea but this isn't the right place for the code. Compare to this class which is achieving a similar goal

https://github.com/hmcts/finrem-case-orchestration-service/blob/master/src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/FinremCallbackRequestFactory.java

Reply:
I am gonna change the caller to use `FinremCallbackRequestFactory` class.